src/to_wiki.js

```diff
@@ -189,17 +189,22 @@ class WikiTextSerializerState {
          }
       }
 
-      // Borrows from src/to_markdown.js from prosemirror-markdown
-      const [_, start, text, end] = node.text.match(/^(\s*)(.*?)(\s*)$/)
-
-      this.out += this.spaces
-      this.out += start
-      this.spaces = end
-
       this.currentlyOpenMarks = this.currentlyOpenMarks.concat(toOpen)
-      this.openMarks(toOpen)
+      if (node.isText) {
+         // Borrows from src/to_markdown.js from prosemirror-markdown
+         const [_, start, text, end] = node.text.match(/^(\s*)(.*?)(\s*)$/)
```

Comment:
I generally like to avoid placeholder variables if I can because it's unclear what the value of that variable is if we have to refactor.